onos_ric_sdk_py/e2.py

```diff
@@ -1,114 +1,38 @@
 # SPDX-FileCopyrightText: © 2021 Open Networking Foundation <support@opennetworking.org>
 # SPDX-License-Identifier: Apache-2.0
 
-import asyncio
+from __future__ import absolute_import
```

Comment:
Perhaps you should keep the previous one for a while like us

Reply:
Good idea. We have a release pushed to PyPI that uses the deprecated communication mechanism so we can always change our code to use that version instead of supporting both in the source code.